### DISCUSSION TITLE
Wildfly Compatibility

### DISCUSSION THREAD

**Discussion by joelcharlebois** [OpenIdentityPlatform/OpenAM-JEE-Agents#32] [Ideas]
Could you consider adding support for one or more of the later versions of the Wildfly application server?

Wildfly 26.x which is Jakarta EE 8 compliant.
Wildfly 31.x which is Jakarta EE10 compliant. 

A Wildfly 31.x compatible version of the JEE policy agent would lend itself to also having a JBoss EAP v8 compatible version, given that JBoss is built on Wildfly.

Thanks and much appreciated.

**Comment by vharseko** [comment]
https://github.com/OpenIdentityPlatform/OpenAM/issues/579

**Comment by joelcharlebois** [comment]
A quick follow-up note of information;

I have successfully deployed and run OpenAM itself into a Wildfly 26.x instance.  

This is due to the Wildfly 26.x version being Jakarta EE 8 compliant; It still uses and supports the javax namespace.  

As such, perhaps as an intermediate step on the way to a future Jakarta EE10 compliant version for Wildfly 31.x and later, a Wildfly 26.x Jakarta EE 8 version of the JEE policy agent, still using the javax namespace, could be created, with perhaps less time and effort.

Best regards.

**Comment by FireBurn** [comment]
I'm stuck on Wildfly 26.1.3 for another application, it's got a known issue with it's embedded Infinispan, they've no intention of creating a 26.1.4 so if you really need to use Wildfly you might want to fix it yourself:

https://issues.redhat.com/browse/WFLY-18878
https://github.com/wildfly/wildfly/pull/17508

If you're setting up OpenAM, I personally use OpenLiberty, it still works with JavaEE 8 whilst also letting you configure for Jakarta 10, granted I've not tried to use the JEE agent with it
